src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
I see your point on this although I am not entirely unconvinced that adding new attributes to the figure rather than using those already provided in `axes` is a good idea.  I would suggest that you just take the first axes in a figure and perhaps print a warning that none of the other axes will be used.

An advantage to keeping the interface as similar to the existing one is that someone could switch between normal (boring) plotting and Google Earth visualization very easily and less error prone.  I also think maintaining the structure of the data is really important even though some of the functionality is not possible with the Google Earth stuff.